packages/graphback-runtime-knex/tests/service/DefaultCRUDServiceTest.ts

```diff
@@ -94,8 +113,17 @@ test('update Todo', async () => {
   expect(todo.text).toEqual('my updated first todo');
 });
 
-test('delete Todo', async () => {
-  const { services } = await setup({ seedData: { todo: [{ text: 'my first todo' }, { text: 'my second todo' }] } })
+test('delete Todo', async (done) => {
+  const { services } = await setup({ seedData: { todo: [{ text: 'my first todo' }, { text: 'my second todo' }] } });
+
+  const subId = await pubSub.subscribe("DELETE_TODO", ({ deletedTodo }) => {
+    expect(deletedTodo).toEqual({
+      id: 2
+    });
```

Comment:
This test could be a good base when discussing https://github.com/aerogear/graphback/issues/1684